### DISCUSSION TITLE
Introducing first Android preview

### DISCUSSION THREAD

**Discussion by jp-weber** [2fast-team/2fast#118] [Announcements]
First public Android preview with some limitations:
- only loading of a data file possible
- after restarting the device, the app loses authorisation to access the configured data file (wip)
  - a workaround is to open the file with the app or
  - reset the app
- limited range of functions
  - no edit of accounts possible
  - no add of new accounts possible

<hr /><em>This discussion was created from the release <a href='https://github.com/2fast-team/2fast/releases/tag/Android-preview-1'>Introducing first Android preview</a>.</em>

**Comment by Th3-A6add0n** [comment]
Finally

**Comment by katastrophe92** [comment]
I will take a look also

**Comment by jp-weber** [comment]
The second preview will be released in the next few days, which will remove some limitations.

**Comment by htmlcoder1562** [comment]
IT'S ON ANDROID???????????????????????????????!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!????????????????????????!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! YAY!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
